recipes-bsp/acpi-tables/files/acpi-tables-load

```diff
@@ -3,8 +3,10 @@
 # You must have configfs enabled for this to work
 
     for TABLE in `ls /kernel/firmware/acpi/ 2>/dev/null`; do
-	dest_dir=$(basename $TABLE .aml)
-	mkdir /sys/kernel/config/acpi/table/$dest_dir
-	cat /kernel/firmware/acpi/$TABLE > /sys/kernel/config/acpi/table/$dest_dir/aml
+        dest_dir=$(basename $TABLE .aml)
```

Comment:
Is this tab vs. space changes? Please, drop.

Reply:
> Is this tab vs. space changes? Please, drop.
Yes

Reply:
OK, so, do them in separate change if needed.